lc0/src/mcts/search.cc

```diff
@@ -652,6 +654,14 @@ Node* Search::PickNodeToExtend(Node* node, PositionHistory* history) {
             remaining_playouts_ < best_node_n - iter->GetNStarted()) {
           continue;
         }
+        // If searchmoves was sent, restrict the search only in that moves
+        if (!limits_.searchmoves.empty() &&
+            std::find(limits_.searchmoves.begin(),
+                      limits_.searchmoves.end(),
+                      iter->GetMove(root_color).as_string()) ==
```

Comment:
It's better to do vice versa, convert searchmoves from string to Move (somewhere before getting into Search class) and then compare.
For example, promotions can be encoded both as "e7e8q" and "e7e8Q", and I'm not sure how knight promotion works too.

Reply:
Ok.